metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/form/FormAssignmentHook.java

```diff
@@ -49,7 +50,7 @@
  */
 @Slf4j
 @Component
-@Singleton
```

Comment:
We don't need these singleton annotations, I'd rather just remove this than change it to a new API